src/main/java/net/tridentsdk/meta/item/ItemDisplayProperties.java

```diff
@@ -56,4 +59,32 @@
     default void setLore(String... lore) {
         setLore(Arrays.asList(lore));
     }
+    
+    /**
+     * Adds lore to the item starting from a specific index.    
+     * @param index Index to begin insertion at. If equal to number of lore, it appends.
+     * @param lore Lore to be added.
+     * @see #addLore(String...)
+     */
+    default void addLore(int index, String... lore) {
+        List<String> curr = lore();
+        if (curr == null)
+            curr = Lists.newArrayList();
+        if (curr.isEmpty() || index == curr.size()) {
```

Comment:
The technical index check can be a hard detail to find. I'd say remove it, and change your implementation of `addLore(String[])`